styles/styles.css

```diff
@@ -1,130 +1,160 @@
+.category {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
   margin: 2rem;
-  position: relative;
+  gap: 1rem;
 }
 
-.home-sale-banner img {
-  height: 400px;
+.category img {
+  height: 250px;
   width: 100%;
+  margin-bottom: 0.5rem;
 }
 
-.home-sale-banner-body {
-  position: absolute;
-  top: 50%;
-  left: 50%;
-  transform: translate(-50%, -50%);
-  -ms-transform: translate(-50%, -50%);
-  padding: 0.5rem 2.5rem;
-  background-color: rgba(243, 243, 249, 0.77);
-  color: var(--primary);
+.category-wrapper {
+  background-color: var(--background);
+  padding: 1rem;
 }
 
-/* new arrivals */
+.category-wrapper:hover {
+  box-shadow: var(--shadow);
+  color: var(--secondary-dark-variant);
+}
+
+/* genre */
 
-.home-new-arrivals-container {
+.genre-container {
   display: grid;
-  grid-template-columns: repeat(2, 1fr);
+  grid-template-columns: repeat(6, 1fr);
+  margin: 2rem;
+  gap: 1rem;
 }
 
-/* media queries */
-@media (max-width: 1275px) {
-  .card-desc {
-    display: none;
-  }
+.genre-container img {
+  height: 210px;
+  width: 100%;
 }
+
+/* choice */
+.icon-container {
+  position: relative;
+}
+
+.card-icon-vertical {
+  right: 0.5rem;
+  top: 0.8rem;
+}
+
+.card-container-vertical {
+  height: 530px;
+  border: 2px solid var(--white);
+}
+
+.card-container-vertical img {
+  padding: 0.5rem;
+  height: 40%;
+}
+
+.choice-container {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  margin: 2rem;
+  justify-items: center;
+}
+
+.card-body-vertical {
+  padding: 0 1rem;
+}
+
+/* media queries */
 @media (max-width: 1210px) {
-  .btn-whishlist {
+  .card-4 {
     display: none;
   }
-}
-
-@media (max-width: 900px) {
-  .home-display-category-container {
+  .choice-container {
     grid-template-columns: repeat(3, 1fr);
   }
-  .home-new-arrivals-container {
-    grid-template-columns: repeat(1, 1fr);
+  .genre-container {
+    grid-template-columns: repeat(3, 1fr);
   }
 }
 
-@media (max-width: 550px) {
-  .home-display-category-container {
+@media (max-width: 800px) {
+  .choice-container {
     grid-template-columns: repeat(2, 1fr);
   }
-  .home-sale-banner-body {
-    padding: 0.5rem 1rem;
+  .genre-container {
+    grid-template-columns: repeat(2, 1fr);
   }
-  .home-new-arrivals-container {
-    grid-template-columns: repeat(1, 1fr);
+  .card-4 {
+    display: block;
   }
 }
 
-@media (max-width: 580px) {
-  .card-container-horizontal {
-    display: block;
+@media (max-width: 550px) {
+  .hero-section-body {
+    padding: 0.5rem 1rem;
+    width: 250px;
   }
-  .card-container-horizontal img {
-    height: 120px;
+  .choice-container {
+    margin: 0.5rem;
   }
-  .card-heading {
-    font-size: 0.75rem;
-    line-height: 1rem;
+}
+
+@media (max-width: 430px) {
+  .choice-container {
+    grid-template-columns: repeat(1, 1fr);
   }
-  .card-price {
-    display: none;
+  .genre-container {
+    grid-template-columns: repeat(1, 1fr);
+  }
+  .category {
+    grid-template-columns: repeat(1, 1fr);
```

Comment:
you can just do, 

.choice-container , 
.genre-container , 
.category {
    grid-template-columns: 1fr;
  }

Reply:
Thank you for the review I'll update it.